src/services/storage/s3Service.ts

```
@@ -0,0 +1,329 @@
+// ── Virus scan hook ───────────────────────────────────────────────────────────
+
+/**
+ * Read the `scan-status` tag from an S3 object.
+ * Returns "pending" | "clean" | "infected" | "unknown".
+ *
+ * In production this tag is written by a Lambda/ClamAV scanner triggered on
+ * s3:ObjectCreated events. The tag acts as the gate for download URL issuance.
+ */
+export async function getObjectScanStatus(
+  objectKey: string,
+): Promise<string> {
```

ESLint

[error] Replace `⏎··objectKey:·string,⏎` with `objectKey:·string`

(prettier/prettier)